.github/workflows/docker.yml

```diff
@@ -0,0 +1,80 @@
+on:
+  schedule:
+    - cron: '0 */1 * * *' # This should run every hour
+
+name: Docker images
+
+jobs:
+  build-images:
+    name: Build & push Docker images
+    runs-on: ubuntu-20.04
+
+    strategy:
+      matrix:
+        runtime:
+          - name: fdo-20-08
```

Comment:
I would rather use `freedesktop-20.08` here, same thing for gnome/kde (for the version)

Reply:
Done